crates/era-downloader/tests/it/list.rs

```diff
@@ -0,0 +1,32 @@
+//! Tests fetching a list of files
+use reqwest::Url;
+use reth_era_downloader::EraClient;
+use std::{
+    hash::{DefaultHasher, Hash, Hasher},
+    path::PathBuf,
+    str::FromStr,
+};
+use test_case::test_case;
+
+#[test_case("https://mainnet.era1.nimbus.team/"; "nimbus")]
+#[test_case("https://era1.ethportal.net/"; "ethportal")]
+#[tokio::test]
+async fn test_getting_file_name_after_fetching_file_list(url: &str) {
```

Comment:
I'm fine with having them in CI for now, but point them to earlier era files on sepolia which should be small files

if this looks stable or is taking a long time we should just mark them as ignore